.github/workflows/CI.yml

```
@@ -0,0 +1,54 @@
+        run: dotnet tool install --global dotnet-coverage
+
+      # 4. Begin SonarCloud analysis
+      - name: SonarCloud Begin Analysis
+        run: |
+          dotnet tool install --global dotnet-sonarscanner
+          dotnet sonarscanner begin /k:"Learnathon-By-Geeky-Solutions_dev-4545" /o:"learnathon-by-geeky-solutions" /d:sonar.token="${{ secrets.SONAR_TOKEN }}" /d:sonar.cs.vscoveragexml.reportsPaths="backend/Employee.API/EmployeeXUnit.Test/coverage.xml"
+
+      # 5. Build the solution
+      - name: Build solution
+        run: dotnet build --no-incremental backend/Employee.API
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)